### ISSUE TITLE
Exhibiting Relationships in the UI

### ISSUE THREAD

**Issue by jyssy** [LD4P/sinopia_editor#2051]
**Visually connecting (even if with just a line or something) Items to Instances to Works, etc**

** Today, we at University of Chicago were looking at something multiples of folks are cataloging in Sinopia (more testing, etc) and we realized that we need a way to see, right off, that an Instance is connected to which Work, or vice versa. Specifically, we wanted to know that this Instance https://trellis.stage.sinopia.io/repository/chicago/a224aa54-bc63-4d63-88e1-2d18c2dae51f is an Instance of this Work: https://trellis.stage.sinopia.io/repository/chicago/a8c8692d-c9bc-42b2-a4b5-4796d4606126.

And vice versa.

We think it would be super useful to add some kind of visual clue to the left of the UI that denotes the relationships between Works, Instances, and Items (perhaps).
**

**We could of course open each Work or Instance to verify it's the one we need, but that will add up to a LOT of time over time.**

So, we think our suggestion would help everyone.

thank you

**Comment by michelleif** [comment]
great idea. what do you think about a UI that would allow you to also **create** relationships visually, like having a blob representing a work and another representing an instance and you drag from one to the other and specify a relationship? or you're cataloging a work and you want to say it's a translation of another work, you find some other related works and drag one to the other?

**Comment by jyssy** [comment]
The question would be whether one would create the relationship visually or if the insertion of a URI would automatically produce the relationship since the ontology itself defines it. Let me ask the UChicago folks and see what they think. But in terms of cataloging workflow, it seems to me, just off-the-cuff,' that by adding the URI for the work, instance, item itself WHILE actually working in the template would be the quickest way to then mark that for the system to produce the visual representation of the relationship. But, again, let me reach out and see what folks think. thank you.

~ Jesse

**Comment by michelleif** [comment]
great! this might be something we could ask @astridu to prototype. i was more thinking of the visual as the _way_ the relationship is created, no need to add URI, instead you search for the thing you want to relate, you get some graphic representation of choices, you drag and drop, you're prompted to choose a property / relationship, and voila

**Comment by jcoyne** [comment]
@michelleif please keep in mind that drag and drop is a real challenge for the blind. We should create interfaces that are accessible to everyone.

**Comment by jcoyne** [comment]
I think this also gets to the problem of this being a general purpose linked data editor. In Bibframe "Work" -> "Instance" -> "Item" is very important to the user, but we don't have a general purpose way of knowing which relationships are important.

**Comment by michelleif** [comment]
thanks @jcoyne good point about accessibility. We should remain open to other accessible UX possibilities for interacting w linked data entities. Relationships will always be important so even in a general purpose LD editor, if there are other ways to construct and see relationships I think that would be useful.

**Comment by jermnelson** [comment]
There are ways to model the relationships between RDF entities using [SKOS](https://www.w3.org/TR/skos-reference/), [OWL](https://www.w3.org/OWL/), or even basic `rdfs:domain` and `rdfs:range`.

**Comment by michelleif** [comment]
@jermnelson I'm just talking about the UX for constructing and viewing those relationships.

**Comment by jyssy** [comment]
@jermnelson Yes, this idea was, originally, just about suggesting the possibility of adding some visual cue that a relationship exists between two entities via URI. thank you.
~ Jesse

**Comment by justinlittman** [comment]
Requirement from #2598: User is able to navigate to related resources (i.e., open them in a new editor tab).

**Comment by michelleif** [comment]
FOLIO has a nice three-pane layout which might offer inspiration for a way to work with multiple resources at once.

**Comment by michelleif** [comment]
#3227 will implement this
